### PR TITLE
add suport for Tuxcoin DonationSubsidy

### PR DESCRIPTION
**Pull request by bleach86**
This is to add support for the Tuxcoin DonationSubsidy hardfork at block 302,400.

### CODE REVIEW

**Comment by crackfoo** [comment]
These changes break charity payments for existing coins using charity.

File: stratum/coinbase.cpp [hunk 1]
```diff
@@ -313,7 +327,17 @@ void coinbase_create(YAAMP_COIND *coind, YAAMP_JOB_TEMPLATE *templ, json_value *
 			strcat(templ->coinb2, "02");
 		}
 		job_pack_tx(coind, templ->coinb2, available, NULL);
-		job_pack_tx(coind, templ->coinb2, charity_amount, script_payee);
+		// job_pack_tx(coind, templ->coinb2, charity_amount, script_payee);
```
Comment (ryan-shaw) [inline stratum/coinbase.cpp]: @bleach86 this is the bits that do the breaking

**Comment by bleach86** [comment]
We have fixed the issue with interference with other coins charity payments

**Comment by tpruvot** [comment]
tx, merged

**Pull request closed without merging**